### DISCUSSION TITLE
Integrated data

### DISCUSSION THREAD

**Discussion by HugoCroizer** [theislab/scvelo#424] [Q&A] [answered]
Hi and thank you for the awesome package. 

I was wondering if you made any progress concerning the analysis of integrated data (like with seurat anchors)? 
We have single cell data from multiple patient integrated together, we think there is a cell population showing the transition between two states, however this population seem to be rare, and just patient by patient there is not enough information. 

Just merging the patient together, it seems each patient for a given gene, as a different activated steady state.

Thank you

**Comment by dawe** [comment]
I would compute the cell graph using bbknn (or harmony?) and calculate moments from that, at least to ensure batches are properly handled.

**Comment by VolkerBergen** [comment, marked as answer]
hi @Dantferno, coming up with a method to handle multiple batches has been rather challenging. What our analysis has shown though, is that from all the methods we've tested, only bbbknn (as @dawe suggested) seemed to be applicable without further extension. That is because it simply adjusts the neighbor graph which is then used to compute moments for un/spliced from the same graph without need of any further transformation of the expression matrices.
